include/fmt/args.h

```diff
@@ -59,6 +59,7 @@ class dynamic_arg_list {
   }
 };
 }  // namespace detail
+FMT_BEGIN_EXPORT
```

Comment:
We only have one class template here so I suggest applying FMT_MODULE_EXPORT to it.